spring-cloud-task-docs/src/main/asciidoc/index.adoc

```diff
@@ -40,10 +42,25 @@ Michael Minella, Glenn Renfro
 // ======================================================================================
 
 include::preface.adoc[]
+ifdef::backend-pdf[]
+<<<
+endif::backend-pdf[]
 include::getting-started.adoc[]
```

Comment:
It is good practice to always have a blank line before an include, in case files don't start with a title